frontend/src/FullPageRoutes/Matching.tsx

```diff
@@ -6,11 +6,59 @@ import {
   makeStyles,
   Avatar,
   IconButton,
+  TextField,
+  Select,
+  MenuItem,
+  Chip,
 } from "@material-ui/core"
+import { Autocomplete } from "@material-ui/lab"
 import { Check, Block, Flag, PlayCircleFilled } from "@material-ui/icons"
-import { request } from "../utils/fetch"
+import { request, queryParams } from "../utils/fetch"
 import ViewVideo from "../components/video/ViewVideo"
 
+type EnqueueFunc<T> = (...newItems: T[]) => void
+
+type MatchControlsType = {
+  matchId: number
+  onAccept: (matchId: number) => void
+  onReject: (matchId: number) => void
+  onReport: (matchId: number) => void
+}
+
+type Match = {
+  id: number
+  first_name: string
+  last_name: string
+  gender: string
+  birth_date: string
+  email: string
+  phone_number: string
+  interests: Interest[]
+  has_intro: boolean
+}
+
+type InterestID = number
```

Comment:
It's not. I was trying to avoid using the `Interest` type in the filter state so it would be easier to encode in the URL. I don't need it anymore but I didn't feel it was worth removing

Reply:
Fair